lms/djangoapps/teams/static/teams/js/views/topic_teams.js

```diff
@@ -17,19 +17,40 @@
                 },
 
                 render: function() {
-                    TeamsView.prototype.render.call(this);
-
-                    if (this.teamMemberships.canUserCreateTeam()) {
-                        var message = interpolate_text(
-                            _.escape(gettext("Try {browse_span_start}browsing all teams{span_end} or {search_span_start}searching team descriptions{span_end}. If you still can't find a team to join, {create_span_start}create a new team in this topic{span_end}.")),
-                            {
-                                'browse_span_start': '<a class="browse-teams" href="">',
-                                'search_span_start': '<a class="search-teams" href="">',
-                                'create_span_start': '<a class="create-team" href="">',
-                                'span_end': '</a>'
-                            }
-                        );
-                        this.$el.append(_.template(teamActionsTemplate, {message: message}));
+                    var self = this,
+                        collectionRefresh;
+                    if (this.collection.isStale || this.teamMemberships.isStale) {
```

Comment:
It might be nice to factor the code that happens if `isStale` into separate methods.